src/containers/EditCourse/index.jsx

```diff
@@ -33,7 +33,7 @@ const mapDispatchToProps = {
 const mergeProps = (stateProps, actionProps, { id }) => ({
   ...stateProps,
   fetchCourseInfo: () => actionProps.fetchCourseInfo(id),
-  fetchCourseOptions: () => actionProps.fetchCourseOptions(id),
+  fetchCourseOptions: () => actionProps.fetchCourseOptions(),
```

Comment:
?

Reply:
I dropped the id argument to the `fetchCourseOptions` action, because now we just hit the list endpoint.